### PR TITLE
rename update

### PR DESCRIPTION
**Pull request by cameronmwall**
Signed-off-by: Cameron Wall <cwall@redhat.com>

### CODE REVIEW

**Comment by cameronmwall** [comment]
issue: https://github.com/stolostron/backlog/issues/18704

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/open-cluster-management/backplane-operator
-  newTag: latest
+  newName: quay.io/cameronmwall/backplane-operator
```
Comment (JakobGray) [inline config/manager/kustomization.yaml:15]: looks dev-y

**Review by JakobGray — COMMENTED** [review]
Dockerfile base images with open-cluster-management should be updated as well

**Review by JakobGray — APPROVED** [review]

**Review by zkayyali812 — APPROVED** [review]

**Pull request merged**